### ISSUE TITLE
Prevent sending data to GAFAM and other thirds party

### ISSUE THREAD

**Issue by lapineige** [YunoHost-Apps/diagramsnet_ynh#6]
Hello,
I recently heard about this issue: by default Draw.io/Diagrams.net send data to thirds parties, including Google, Microsoft and Trello. That might lead to some privacy issues. See:https://shelter.moe/@Neil/103928506546498078 (I can translate it if needed)

Is this problem tackled by this package ?
If not, it would be good to fix it by default, or to give an option to disable this during installation (as turning it to offline mode can be a bit limiting and not wanted for some users).

Thanks :)

**Comment by Gofannon** [comment]
> Is this problem tackled by this package ?

No. I just tested on my server and it sends data to GAFAM+.

Thanks for the tip on how to fix it. At first glance, it seems easy to fix. I'm wondering on the long term if the fix will need to be adapted.

**Comment by Gofannon** [comment]
If you can, could you test this version? It should fix it.

```bash
sudo yunohost app install https://github.com/YunoHost-Apps/diagramsnet_ynh/tree/testing --debug
or
sudo yunohost app upgrade diagramsnet -u https://github.com/YunoHost-Apps/diagramsnet_ynh/tree/testing --debug
```

**Comment by Gredin67** [comment]
Guess this issue is solved #7
